Description: Move $(LDFLAGS), so that -Wl,--as-need can be effectively passed to linker.

Index: gmic-1.3.2.9/src/Makefile
===================================================================
--- gmic-1.3.2.9.orig/src/Makefile	2009-11-09 17:05:10.000000000 +0100
+++ gmic-1.3.2.9/src/Makefile	2009-11-09 17:05:12.000000000 +0100
@@ -158,7 +158,7 @@
 gmic_gimp.o: gmic.cpp gmic_def.h
 	$(CC) -o gmic_gimp.o -c gmic.cpp -Dgmic_gimp -Dgmic_minimal $(CFLAGS)
 gmic_gimp : gmic_gimp.o gmic_gimp.cpp
-	$(CC) -o gmic_gimp gmic_gimp.cpp gmic_gimp.o `gimptool-2.0 --cflags` `gimptool-2.0 --libs` $(CFLAGS) $(LDFLAGS)
+	$(CC) -o gmic_gimp gmic_gimp.cpp gmic_gimp.o $(LDFLAGS) `gimptool-2.0 --cflags` `gimptool-2.0 --libs` $(CFLAGS)
 ifeq ($(STRIP_EXE),1)
 	strip gmic_gimp
 endif
